mne/decoding/tests/test_time_gen.py

```diff
@@ -333,5 +335,11 @@ def test_decoding_time():
     assert_equal("<TimeDecoding | fitted, start : -0.200 (s), stop : 0.499 "
                  "(s), predicted 14 epochs,\n scored (accuracy_score)>",
                  '%s' % tg)
+    # Test with regressor
+    clf = SVR()
+    cv = KFold(len(epochs))
+    y = np.random.rand(len(epochs))
+    tg = TimeDecoding(clf=clf, cv=cv)
```

Comment:
The other important problem here was that a parameter was not passed. It seems the auto-detection is a separate issue.